exporter/elasticsearchexporter/model.go

```diff
@@ -41,6 +41,9 @@ const (
 
 func (m *encodeModel) encodeLog(resource pcommon.Resource, record plog.LogRecord, scope pcommon.InstrumentationScope) ([]byte, error) {
 	var document objmodel.Document
+	if record.Timestamp().AsTime().UnixNano() == 0 {
+		record.SetTimestamp(record.ObservedTimestamp())
```

Comment:
You are modifying the log record here, but I don't think the exporter declares that it mutates data? If I remember correctly, this might cause a panic.

I suggest to implement this without writing to the original log record. Perhaps declare a local variable instead.

Reply:
make sense. I will update here